.travis.yml

```diff
@@ -11,8 +11,14 @@ env:
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
 addons:
   postgresql: '9.4'
-install: bin/setup
+install: 
+  - sudo bin/qpid_install.sh
+  - gem install specific_install
+  - gem specific_install https://github.com/xlab-si/qpid_proton_gem
```

Comment:
Is this effectively a straight extraction of  https://github.com/apache/qpid-proton/tree/master/proton-c/bindings/ruby ?